back_end/README.md

```diff
@@ -0,0 +1,20 @@
+# Back End Challenge
+
+Your challenge is to create a simple API application using Ruby on Rails (or the web development framework of your choice). Being a SVOD service, we have film data that we need to surface through an API. Your challenge is build a small sample endpoint for retrieving film data and submitting film ratings.
```

Comment:
Missing a preposition the last sentence. Should read "Your challenge is to..." Also, we repeat "Your challenge is to" twice in this paragraph. Maybe change the paragraph to read:
```
Being a SVOD service, we have film data that we need to surface through an API. Your challenge is to create a simple API application using Ruby on Rails (or the web development framework of your choice). 
```

I think the instructions below list the two main functions we're looking for.